src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -2,41 +2,62 @@ import { NextFunction, Request, Response } from "express";
 import { Errors } from "config/errors";
 import { Subscription } from "~/src/models/subscription";
 import { GitHubServerApp } from "~/src/models/github-server-app";
+import { getGitHubApiUrl } from "utils/get-github-client-config";
+import axios from "axios";
+import Logger from "bunyan";
 
 export const GithubCreateBranchOptionsGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
-	const { jiraHost } = res.locals;
-	const { issue_key: key } = req.query;
+	const { jiraHost, gitHubAppConfig } = res.locals;
+	const { issueKey } = req.query;
+	const { githubToken } = req.session;
+	const gitHubAppId = res.locals.gitHubAppId || gitHubAppConfig?.gitHubAppId;
```

Comment:
`gitHubAppConfig` object is only available in `/github` route.  In this case, it will be undefined and vaildated against GH cloud everytime.